### DISCUSSION TITLE
blog/devlog-1-new-stuff/

### DISCUSSION THREAD

**Discussion by giscus** [modulocity/automationstationgame.com#3] [Comments]
# blog/devlog-1-new-stuff/

https://www.automationstationgame.com/blog/devlog-1-new-stuff/

**Comment by Introscopia** [comment]
I am absolutely enchanted by what I'm seeing here! Can't wait to see the little robo-arms animated. Oh, and I seriously approve of taking rubber seriously as a basic crafting material!

**Reply by ScottJDaley** [reply]
Thanks for the kind words! Any thoughts on how blubber should be turned into rubber?

**Reply by krooq** [reply]
Smelter is not that far from reality.
If it is a basic material I imagine you will want a simple process to prepare it.
It definitely falls into the "liquid" processing category which is a whole different kettle of fish.
My only thought is some kind of "vat" that heats and has a nozzle that extrudes.
So its sort of like a liquid equivalent of a smelter. Almost like a soft serve ice-cream machine XD

Excited to see what you come up with.

**Reply by ScottJDaley** [reply]
Yeah that does sound kind of similar to how the crucible and casting already works for smelting ore. Perhaps the crucible could be reused for rubber, but it would just require a different kind of mold to pour the hot rubber into. 

It would also be neat if the rubber had to be pressed into sheets by some nice chunky roller contraption. Apparently its called calendering in technical terms. However, I think this is too complicated of a process for a basic material in the game.
